discord/interactions.py

```diff
@@ -103,6 +104,8 @@ class Interaction:
         for 15 minutes.
     data: :class:`dict`
         The raw interaction data.
+    locale: Optional[:class:`Locale`]
+        The locale of the interaction.
```

Comment:
It would be good to specify that this is the user's locale.

Reply:
Will do! I completely missed the `guild_locale` so I'll add that too.

Which locale should I default to if none is given (as in, in the ping type interaction) or what would be the best way to handle that.

Reply:
I'm not sure defaulting to a specific language would be the best here. `try_enum` could return an unknown enum with a value of `None` if the locale wasn't provided, which should be okay considering it will be provided most of the time. You could have a note in the docs saying that it will be an unknown value if the interaction was a ping interaction, or something similar.

Reply:
I see, I tried passing None but,
```python
Traceback (most recent call last):
  File "--------\discord.py\main.py", line 2, in <module>
    print(try_enum(Locale, None))
TypeError: __str__ returned non-string (type NoneType)
```
So I went back to defaulting 'en-US' which, I don't really like, but it's what `Guild.preferred_locale` does too.